.github/workflows/ci.yml

```diff
@@ -85,3 +85,41 @@ jobs:
         with:
           severity: error
           ignore_paths: third-party
+  dokken-recipes:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os:
+          - alinux2
+          - centos7
+          #- centos8
+          - ubuntu18
+          - ubuntu20
+        suite: [ 'base' ]
+      fail-fast: false
+    steps:
+      - name: Check out code
+        uses: actions/checkout@main
+      - name: Install Chef
+        uses: actionshub/chef-install@main
+      - name: Test-Kitchen Converge
+        uses: actionshub/test-kitchen@main
+        with:
+          suite: ${{ matrix.suite }}
+          os: ${{ matrix.os }}
+          action: converge
```

Comment:
Should be `test`, not `converge` I guess.

Reply:
I removed it, by default it will execute the test action, addressed here: https://github.com/aws/aws-parallelcluster-cookbook/pull/1648